addons/mail/static/tests/qunit_suite_tests/components/discuss_pinned_tests.js

```diff
@@ -146,7 +146,6 @@ QUnit.test('sidebar: unpin channel from bus', async function (assert) {
             'channel_type': 'channel',
             'id': mailChannelId1,
             'name': "General",
```

Comment:
channel unpin does not send anything else in python than the id, you might as well remove all the other keys from there